models/engine/db_storage.py

```diff
@@ -74,3 +74,30 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """
+        Retrieves an object based on the class name and its ID,
+        None if not found
+
+        cls (class): name of the class
+        id  (str): id of the object
+        """
+        if not cls or not id:
+            return None
+
+        objects = self.all(cls).values()
+        for obj in objects:
```

Comment:
What does obj mean ?, you should put a more significant name to the variable

Reply:
Yeah you're right.